apps/vaults/utils/actions/stakingRewards.tsx

```diff
@@ -0,0 +1,52 @@
+import {ethers} from 'ethers';
+import {STAKING_REWARDS_ZAP_ADDRESS} from '@yearn-finance/web-lib/utils/constants';
+import {handleTx} from '@yearn-finance/web-lib/utils/web3/transaction';
+
+import STAKING_REWARDS_ABI from '../abi/stakingRewards.abi';
+import STAKING_REWARDS_ZAP_ABI from '../abi/stakingRewardsZap.abi';
+
+import type {BigNumber} from 'ethers';
+import type {TAddress} from '@yearn-finance/web-lib/types';
+import type {TTxResponse} from '@yearn-finance/web-lib/utils/web3/transaction';
+
+export async function stake(
+	provider: ethers.providers.JsonRpcProvider,
+	accountAddress: TAddress,
+	stakingAddress: TAddress,
+	amount: BigNumber
+): Promise<TTxResponse> {
+	const signer = provider.getSigner(accountAddress);
+	const stakingRewardsContract = new ethers.Contract(stakingAddress, STAKING_REWARDS_ABI, signer);
+	return await handleTx(stakingRewardsContract.stake(amount));
```

Comment:
Follow up on https://github.com/yearn/yearn.fi/pull/65/files#r1167806331, as we're not using the returned value there, we could do the following:

```suggestion
	await handleTx(stakingRewardsContract.stake(amount));
	return;
```